lib/tools/system-calls.js

```diff
@@ -438,7 +438,9 @@ systemCallMethods.launchAVD = async function (avdName, avdArgs, language, countr
   let proc = new SubProcess(emulatorBinaryPath, launchArgs);
   await proc.start(0);
   proc.on('output', (stdout, stderr) => {
-    log.info(`[AVD OUTPUT] ${stdout || stderr}`);
+    for (let line of (stdout || stderr || '').split('\n').filter(Boolean)) {
```

Comment:
`(stdout || stderr || ''`: does this mean that the situation when stdout and stderr are both defined is not possible?